commands/command_divelist.cpp

```diff
@@ -404,34 +391,33 @@ AddDive::AddDive(dive *d, bool autogroup, bool newNumber)
 	setText(Command::Base::tr("add dive"));
 	// By convention, d is a pointer to "displayed dive" or a temporary variable and can be overwritten.
 	d->maxdepth.mm = 0;
-	d->dc.maxdepth.mm = 0;
+	d->dcs[0].maxdepth.mm = 0;
```

Comment:
This doesn't seem to be an issue here, because `addDive()` is only called on dives with a single DC. For consistency, this perhaps should be changed. Or perhaps we should use `force_fixup_dive()`?

In any case, I tried to change behavior as little as possible, so let's do that kind of changes afterward.